### ISSUE TITLE
Add other permissions.

### ISSUE THREAD

**Issue by DrizzyySP** [NamelessMC/Nameless#2485]
### Describe the feature

**I would like it to be possible to determine which groups can have access to each moderation function (edit, delete, block topic, etc)**, that is, allow only a certain group to delete a topic from a certain area of the forum for example.

And if the group the user is in doesn't have permission to delete topics in that area of the forum, the delete button becomes invisible.

**Example:**
- https://i.imgur.com/wZAZnwR.png

Apart from the permissions for each forum area that already exists, these other permissions would be added.
**In short, separate each moderation option into separate permissions to improve overall forum moderation and prevent team members from doing things they shouldn't do on the forum.**

**Comment by DrizzyySP** [comment]
...

**Comment by Aberdeener** [comment]
> ...

What's the purpose of this comment? 🙂

This is on our backlog to do. Thanks for the suggestion! We will get to it when we get to it

**Comment by DrizzyySP** [comment]
Oh, thank you!
